src/LiveComponent/src/DefaultActionTrait.php

```diff
@@ -15,24 +15,11 @@
  * @author Kevin Bond <kevinbond@gmail.com>
  *
  * @experimental
- *
- * @internal
  */
-final class DefaultComponentController
+trait DefaultActionTrait
 {
-    private object $component;
-
-    public function __construct(object $component)
-    {
-        $this->component = $component;
-    }
-
     public function __invoke(): void
     {
-    }
-
-    public function getComponent(): object
-    {
-        return $this->component;
+        // noop - this is the default action
```

Comment:
Maybe some docs above it?

```
The "default" action for a component.

This is executed when your component is being re-rendered, but no custom action
is being called. You probably don't want to do any work here because this method
is *not* executed when a custom action is triggered. 
```

Reply:
Haha thanks, as I was writing the doc I thought: "don't put too much effort here, Ryan will suggest something better".